.github/workflows/ci.yml

```diff
@@ -5,9 +5,6 @@ on:
     branches:
     - master
   pull_request:
-  # Run daily at 0:01 UTC
-  schedule:
-  - cron:  '1 0 * * *'
```

Comment:
why was this removed? You want to know if your current versions break against released dependencies even if `--pre` builds are breaking/passing.

Reply:
I'm essentially assuming that if we get to catch upstream breakage ahead of time, we should never see anything break with releases, but that may be too optimistic of an assumption. I wouldn't oppose restoring this.

Reply:
I disagree with this logic, I’m gonna add it back.